pegen/pegen.c

```diff
@@ -304,6 +304,49 @@ endmarker_token(Parser *p)
     return expect_token(p, ENDMARKER);
 }
 
+static const char *reserved_keywords[] = {
+    "and",
+    "as",
+    "assert",
+    "async",
+    "await",
+    "break",
+    "class",
+    "continue",
+    "def",
+    "del",
+    "elif",
+    "else",
+    "except",
+    "finally",
+    "for",
+    "from",
+    "global",
+    "if",
+    "import",
+    "in",
+    "is",
+    "lambda",
+    "nonlocal",
+    "not",
+    "or",
+    "pass",
+    "raise",
+    "return",
+    "try",
+    "while",
+    "with",
+    "yield"
+};
+static int // bool
+_is_reserved_keyword(char *s) {
+    for (int i = 0; i < 32; i++) {
+        if (!strcmp(s, reserved_keywords[i]))
```

Comment:
I find the spelling `!strcmp(a, b)` confusing, since `!` makes my brain say *not*. Even though it's more typing, can you change to `strcmp(a, b) == 0`? I'm sure the C compiler optimizes it the right way.